prometheus/config/prometheus.yml

```
@@ -2,6 +2,18 @@ global:
 scrape_configs:
+  - job_name: "prometheus"
+    static_configs:
+      - targets: ["localhost:9090"]
+
+  - job_name: "docker"
+    static_configs:
+      - targets: ["host.docker.internal:9323"]
+
+  - job_name: 'cadvisor'
+    static_configs:
+      - targets: [ 'cadvisor:8080' ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)